homeassistant/components/weatherflow/strings.json

```diff
@@ -79,6 +79,14 @@
       "wind_lull": {
         "name": "Wind lull"
       }
+    },
+    "event": {
+      "lightning_strike_event": {
+        "name": "Lightning strike"
+      },
+      "rain_start_event": {
```

Comment:
I'm wondering if we should use `precip_start` instead of rain.

https://weatherflow.github.io/Tempest/api/ws.html < they use precip here